packages/react-components/react-drawer/library/src/shared/drawerMotions.ts

```diff
@@ -14,22 +17,37 @@ const durations: Record<NonNullable<DrawerBaseProps['size']>, number> = {
   full: motionTokens.durationUltraSlow,
 };
 
+function getPositionTransform(
```

Comment:
I guess you can make it more DRY ;) 

```suggestion
function getPositionTransform(
  position: DrawerBaseProps['position'],
  sizeVar: string,
  dir: FluentProviderContextValue['dir'],
)  {
  const value = dir === "rtl" ?  `var(${sizeVar}), 0, 0` : `calc(var(${sizeVar}) * - 1), 0 , 0`; 
  
  if (position !== "bottom") {
    return `translate3d(${value})`;
  }
  
  return `translate3d(0, var(${sizeVar}), 0)`;
}
```